scd30/scd30.h

```diff
@@ -39,6 +39,11 @@
 extern "C" {
 #endif
 
+/**
+ * The data must be received during the data ready window which is 100ms.
```

Comment:
```suggestion
 * The data must be received during the data ready window which is 100ms after the measurement was taken.
   The window might be slightly affected by the sensor's voltage.
```

Reply:
> The window might be slightly affected by the sensor's voltage.

There are probably D2D variations, voltage and temperature dependencies for the clock speed of the sensor.

Reply:
Should we just leave it off? I'm not sure how strong the effects are and if the users should care about.

Reply:
sure